update.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+msg () {
+	1>&2 echo $*
+}
+
+# check that we are in a prototype
+if ! grep -q govuk-prototype-kit package.json 2> /dev/null; then
+	msg 'ERROR you must run update.sh in a folder containing a GOV.UK Prototype Kit installation'
+	msg 'Exiting'
+	exit 1
+fi
+
+# make the update folder
+mkdir -p update
+# stop git from committing the update folder
+echo '*' > update/.gitignore
+
+cd update
+
+# download the latest Prototype Kit zip file
+if ! ls govuk-prototype-kit*.zip > /dev/null 2>&1; then
+	msg 'Downloading latest version of GOV.UK Prototye Kit...'
+	curl -LJO https://govuk-prototype-kit.herokuapp.com/docs/download
+	msg 'Done'
+fi
+
+# choose the zip file with the largest version number, use this to update from
+ZIP_FILE=$(ls govuk-prototype-kit*.zip | sort -V | tail -n1)
+ZIP_ROOT=${ZIP_FILE//.zip}
+OLD_VERSION=$(cat ../VERSION.txt)
+NEW_VERSION=${ZIP_ROOT#govuk-prototype-kit-}
+
+# unzip the zip file into the 'update' folder
+if [ ! -d "$ZIP_ROOT" ]; then
```

Comment:
We should use the newer version:

```suggestion
if [[ ! -d "$ZIP_ROOT" ]]; then
```

Reply:
Interesting, I usually reach for `[` first because it's POSIX, what's the advantage of using `[[`?

Reply:
It's something that I've been using for so long I can't remember the original reasons, but https://unix.stackexchange.com/a/306115 and https://unix.stackexchange.com/a/306141 have my back with some of the benefits.

I believe part of it is that it's more fully featured, and has a bit more implemented.

As we're using bash, there's no reason to rely on POSIX, I'd say